be/src/exprs/bitmap_function.h

```diff
@@ -99,6 +99,8 @@ class BitmapFunctions {
     static BigIntVal bitmap_max(FunctionContext* ctx, const StringVal& str);
     static StringVal bitmap_subset_in_range(FunctionContext* ctx, const StringVal& src, 
                                             const BigIntVal& range_start, const BigIntVal& range_end);
+    static StringVal sub_bitmap(FunctionContext* ctx, const StringVal& src, 
```

Comment:
Without implementation?

Reply:
bitmap_function.cpp have implementation
After resolving the conflict, format is out of order.